packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -261,6 +276,16 @@ export interface BasicCrawlerOptions<Context extends CrawlingContext = BasicCraw
      */
     statusMessageLoggingInterval?: number;
 
+    /**
+     * Allows overriding the default status message. When the callback returns `null` or `undefined`, the default message will be used as a fallback.
+     */
+    statusMessageCallback?: StatusMessageCallback;
+
+    /**
+     * Allows overriding the default status message. When the callback returns `null` or `undefined`, the default message will be used as a fallback.
+     */
+    statusMessageLogLevel?: LogLevel.DEBUG | LogLevel.INFO | LogLevel.WARNING | LogLevel.ERROR;
```

Comment:
I wonder if these are really needed. If you need to customize it, you can set your own setInterval and disable the Crawlee messages (set huge interval?). Because most likely, if you want to make this good, you would want to make the log level depend on the state of the stats (which is what Crawlee is doing with the high error rate) so setting one on crawler level is not that useful

Reply:
True, what about changing the callback so it calls the `setStatusMessage()`?

Reply:
That way you can also disable the status periodic messages by not calling it inside the callback.